src/org/thoughtcrime/securesms/LogViewFragment.java

```diff
@@ -87,14 +94,49 @@ public String getLogText() {
 
   public Float getLogTextSize() { return logPreview.getTextSize(); }
 
-  public void setLogTextSize(Float Textsize) {
-    logPreview.setTextSize(TypedValue.COMPLEX_UNIT_PX, Textsize);
+  public void setLogTextSize(Float textSize) {
+    logPreview.setTextSize(TypedValue.COMPLEX_UNIT_PX, textSize);
   }
 
   public void scrollDownLog() { logPreview.setSelection(logPreview.getText().length()); }
 
   public void scrollUpLog() { logPreview.setSelection(0); }
 
+  public boolean saveLogFile() throws NoExternalStorageException {
```

Comment:
Is NoExternalStorageException thrown for real? It looks like you caught it in the try catch clause

Reply:
You may be right. I tried different approaches and this might be a leftover. I'll give it a try.